app/code/Magento/Contact/Model/Contact.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Magento\Contact\Model;
+
+use Magento\Contact\Api\ContactInterface;
+use Magento\Contact\Api\Data\ContactFormInterface;
+use Magento\Framework\DataObject;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Serialize\SerializerInterface;
+
+/**
+ * Class Contact
+ * @package Phoenix\Contact\Model
+ */
+class Contact implements ContactInterface
+{
+    /**
+     * @var MailInterface
+     */
+    private $mail;
+    /**
+     * @var SerializerInterface
+     */
+    private $serializer;
+
+    /**
+     * Contact constructor.
+     * @param MailInterface $mail
+     * @param SerializerInterface $serializer
+     */
+    public function __construct(
+        MailInterface $mail,
+        SerializerInterface $serializer
+    ) {
+        $this->mail = $mail;
+        $this->serializer = $serializer;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function send(ContactFormInterface $data)
```

Comment:
Need to consider the possibility to send a list of messages. Does it make sense from a usability perspective?